ansible/roles/master/templates/kubectl.kubeconfig.j2

```diff
@@ -5,7 +5,7 @@ preferences: {}
 clusters:
 - cluster:
     certificate-authority-data: {{ kube_ca_cert|b64encode }}
-    server: https://{{ groups['masters'][0] }}:{{ kube_master_api_port }}
+    server: https://{{ kube_apiserver_endpoint }}:{{ kube_master_api_port }}
```

Comment:
I don't think we can remove groups['masters'][0]. I believe you need to set the kube_apiserver_vip using a conditional. Ex> if masters length > 1, then set the vip. Make sense?